src/java.base/share/classes/jdk/internal/jimage/ImageLocation.java

```diff
@@ -44,7 +45,126 @@ public class ImageLocation {
     public static final int ATTRIBUTE_OFFSET = 5;
     public static final int ATTRIBUTE_COMPRESSED = 6;
     public static final int ATTRIBUTE_UNCOMPRESSED = 7;
-    public static final int ATTRIBUTE_COUNT = 8;
+    public static final int ATTRIBUTE_PREVIEW_FLAGS = 8;
```

Comment:
Please add a comment above these constants that the values are defined in ImageFile.hpp.